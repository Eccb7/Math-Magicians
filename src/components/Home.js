import React from 'react';

const Home = () => (
  <h2 className="home-h">
    WELCOME MATHEMATICIAN.
    <>
      <p className="home-p">
        This is an app wholly to help you with your calculation.
        Ready? Then get to the next page.

      </p>
    </>
  </h2>
);

export default Home;
