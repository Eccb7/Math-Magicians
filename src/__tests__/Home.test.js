// Home.test.js
import React from 'react';
import { render } from '@testing-library/react';
import Home from '../components/Home';

test('renders Home component correctly', () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
