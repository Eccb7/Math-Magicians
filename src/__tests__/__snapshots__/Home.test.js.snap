// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home component correctly 1`] = `
<div>
  <h2
    class="home-h"
  >
    WELCOME MATHEMATICIAN.
    <p
      class="home-p"
    >
      This is an app wholly to help you with your calculation. Ready? Then get to the next page.
    </p>
  </h2>
</div>
`;
